app/models/customer.rb

```diff
@@ -1,18 +1,21 @@
 class Customer < ApplicationRecord
   has_many :invoices
   has_many :transactions, through: :invoices
+
+  validates :first_name, presence: true
+  validates :last_name, presence: true
```

Comment:
These can be chained together like `validates_presence_of :first_name, :last_name`

Reply:
Oh good idea!